messages/edge_function/messages.go

```diff
@@ -39,13 +26,13 @@ var (
 	DescribeFlagOut          = "Exports the output to the given <file_path/file_name.ext>"
 	DescribeFlagFormat       = "Changes the output format passing the json value to the flag"
 	DescribeFlagWithCode     = "Displays the Edge Function's code; disabled by default"
-	DescribeHelpFlag         = "Displays more information about the describe edge-function command"
+	DescribeHelpFlag         = "Displays more information about the describe command"
```

Comment:
it should be: `describe edge-function command` according to our conversation with @gabriel-azion

Reply:
```suggestion
	DescribeHelpFlag         = "Displays more information about the describe edge-function command"
```

Reply:
I'm unsure how it will interfere with the changelog if we use this kind of suggestion here. I'm leaving those anyway, if it's best to update the info in the local branch, then resolve them.

Reply:
I'm not sure how it works with the changelog, to be honest hehe